lib/cli.js

```diff
@@ -1,3 +1,10 @@
+try {
+  const { enableCompileCache } = require('node:module')
+  if (enableCompileCache) {
+    enableCompileCache()
+  }
+} catch (e) { /* empty */ }
```

Comment:
To be safe, I followed the same implementation on `tsc`, ref: https://github.com/microsoft/TypeScript/blob/ef802b1e4ddaf8d6e61d6005614dd796520448f8/Herebyfile.mjs#L332-L337

Reply:
What I should change to make that code ignored?

Reply:
```
/* istanbul ignore next */
```